app/views/index.njk

```diff
@@ -39,7 +39,7 @@
   <div class="nhsuk-grid-row nhsuk-promo-group">
     <div class="nhsuk-grid-column-one-half nhsuk-promo-group__item">
       <div class="nhsuk-promo nhsuk-promo--small">
-        <a class="nhsuk-promo__link-wrapper" href="/#page">
+        <a class="nhsuk-promo__link-wrapper" href=" https://aalto.digital.nhs.uk/#/runView/?viewId=5821ac03-fe21-46ca-b9d1-7cb80a67f9bd&library=a175a50c-6415-4f87-8da6-04f28c690f48">
```

Comment:
Very pedantic of me, but empty white space at the start of `href` value.

Reply:
No thats fine its incorrect and we should make the change.